tests/lib/fixtures.py

```diff
@@ -179,6 +181,16 @@ def mock_default_journal_path(temp_dir):
     }
 
 
+@fixture
+def mock_get_templates_path(temp_dir):
```

Comment:
This mock does not work correctly.

Reply:
Since the method is being called in `controller.py`, you can get it to work by patching `jrnl.controller.get_templates_path` instead of `jrnl.config.get_templates_path`

Reply:
You're right. I misunderstood how this worked. Thanks for the help!